activestorage/app/models/active_storage/blob/analyzable.rb

```diff
@@ -42,6 +42,11 @@ def analyzed?
     analyzed
   end
 
+  # Returns true if the blob's analyzer isn't +ActiveStorage::Analyzer::NullAnalyzer+.
+  def analyzable?
+    analyzer_class != ActiveStorage::Analyzer::NullAnalyzer
```

Comment:
Don’t think we’d need mention NullAnalyzer again for this. Can we split out the detect in analyzer_class or move the NullAnalyzer reference to analyzer?